src/game/crypto/blowfish.cpp

```diff
@@ -20,1057 +21,137 @@
 #include <captainslog.h>
 #include <string.h>
 
+// clang-format off
 const uint32_t BlowfishEngine::s_Blowfish_p_init[BF_ROUNDS + 2] = {
```

Comment:
While we are messing with this code, change the name to s_PInit, I think having Blowfish in the name is overkill as its scoped in the class namespace anyhow.